src/test_autocomplete.py

```diff
@@ -0,0 +1,29 @@
+"""Test module for Autocomplete class."""
+
+TEST_LIST = [
+    'slags',
+    'slain',
+    'slake',
+    'slalom',
+    'slam',
+    'slims',
+    'slimy',
+]
+
+
+def setup_autocompleter(num_results=5):
+    from autocomplete import Autocomplete
+    a = Autocomplete(TEST_LIST, num_results)
+    return a
+
+
+def test_num_results():
+    a = setup_autocompleter()
+    results = a('sla')
+    assert len(results) == 5
```

Comment:
I'd like this to be a bit more complete.  You should verify that if you try to autocomplete something that isn't there you get none, that if there aren't more than the max number of completions, you get the number of results that actually exist, etc.